dashboard/src/actions/charts.ts

```diff
@@ -60,16 +59,6 @@ const allActions = [
 
 export type ChartsAction = ActionType<typeof allActions[number]>;
 
-function dispatchError(dispatch: Dispatch, err: Error) {
-  if (err.message.match("could not find")) {
-    dispatch(errorChart(new NotFoundError(err.message)));
-  } else if (err.message.match("Unable to validate user")) {
-    dispatch(errorChart(new ForbiddenError(err.message)));
-  } else {
-    dispatch(errorChart(err));
-  }
-}
-
```

Comment:
this was a remnant of the 1.x implementation where the error type was read by the ErrorSelector.

Reply:
why would we? I mean, both are 404 errors, it would be difficult to differ between those. Also, the body of the error is still shown to the user so it's not that we are hiding anything.

Reply:
right, it's true that it can be a Forbidden error but in any case, we are not using that distinction now